specification/blueprint/resource-manager/Microsoft.Blueprint/Blueprint/readme.md

```diff
@@ -61,7 +61,6 @@ This is not used by Autorest itself.
 swagger-to-sdk:
   - repo: azure-sdk-for-net
   - repo: azure-sdk-for-go
-  - repo: azure-sdk-for-js
```

Comment:
why do we remove JS SDK generation for this service?

Reply:
This package has never been released.